achievements/contribubit.achievement.js

```diff
@@ -0,0 +1,24 @@
+var _ = require('lodash');
+var console = process.console;
+
+var TheGodfatherConsigliere = {
+	name: 'The Godfather Consigliere',
+	check: function(pullRequest, shall) {
+		if (_.result(pullRequest, 'organization.username') === 'Kibibit') {
+			var achievement = {
+				avatar : 'images/achievements/thegodfather.achievement.jpg',
```

Comment:
this name should be the same name as the achievement filename

Also, I prefer if filename and achievement name are synonym with one another.

I know there are achievements already that don't follow that rule, but we need to start following it